### DISCUSSION TITLE
[预览]v1.12.17

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2254] [分享 / Share]
- 试着在番剧区恢复了一些快捷键. (#1992)

(没错, 就这一条更新, 之后可能再热更一些)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.17-preview'>[预览]v1.12.17</a>.</em>

**Comment by maocd008** [comment]
这个版本自动宽屏失效了，我换回1.12.16正式版就可以自动宽屏

**Reply by Geeyun-JY3** [reply]
要说失效 v1.12.16 也有说失效的：#2225

**Reply by the1812** [reply]
v1.12.17 有改动这一块, 但没完全改好(
